### PR TITLE
Delomboking

### PR DESCRIPTION
**Pull request by marcingrzejszczak**

### CODE REVIEW

**Comment by adriancole** [comment]
thanks for the brute force!

File: README.adoc [hunk 1]
```diff
@@ -114,41 +114,6 @@ following command:
 The generated eclipse projects can be imported by selecting `import existing projects`
 from the `file` menu.
```
Comment (dsyer) [inline README.adoc]: I guess you didn't see that?

**Comment by spencergibb** [comment]
Why delombok?

**Comment by marcingrzejszczak** [comment]
Will create a new PR in some time

**Pull request closed without merging**